### PR TITLE
Added gesture to switch sensor for better Xiaomi magic cube support

### PR DESCRIPTION
**Pull request by FrengerH**
I added the gesture property to the switch sensor in preparation of issue [https://github.com/home-assistant/home-assistant/issues/28572](https://github.com/home-assistant/home-assistant/issues/28572)

### CODE REVIEW

**Comment by Kane610** [comment]
Sweet! Ill look into this within a few days

**Comment by FrengerH** [comment]
Thanks, here you can already see the changes on HA side if you're interested [https://github.com/home-assistant/home-assistant/compare/dev...FrengerH:use-gesture-in-event-for-magic-cube](url)

**Comment by Kane610** [comment]
The link doesn't work

**Comment by FrengerH** [comment]
My bad. The links are fixed now.

File: tests/test_sensors.py [hunk 1]
```diff
@@ -520,6 +521,37 @@
     assert sensor.uniqueid == "00:17:88:01:02:0e:32:a3-02-fc00"
 
 
+async def test_switch_sensor_cube():
+    """Verify that temperature sensor works."""
```
Comment (Kane610) [inline tests/test_sensors.py]:
```suggestion
    """Verify that cube device works."""
```

**Pull request merged**